### PR TITLE
Color with Hex

### PR DESCRIPTION
**Pull request by codecaffeine**
The biggest difference with URBNConvenience is that we’re using Ints rather than String.

``` swift
UIColor(rgb: 0xFF0000)
UIColor(rgba: 0xFF0000FF)
UIColor(redInt: 0xFF, greenInt: 0x00, blueInt:0xFF)
UIColor(redInt: 0xFF, greenInt: 0x00, blueInt:0xFF, alphaInt: 0xFF)
```

The first two options can overflow (Int.max > 0xFFFFFF), in that case, it’ll return black.
The last two options is tied to UInt8, it it can only accept `0x00` - `0xFF` for the values.

### CODE REVIEW

**Comment by Dutcher** [comment]
Looks good 👍

File: Example/Pods/Target Support Files/Pods-URBNSwiftyConvenience_Tests/Pods-URBNSwiftyConvenience_Tests-acknowledgements.markdown [hunk 1]
```diff
@@ -1,3 +1,26 @@
 # Acknowledgements
 This application makes use of the following third party libraries:
+
+## URBNSwiftyConvenience
+
+Copyright (c) 2017 Lloyd <lloyd1nspace@users.noreply.github.com>
```
Comment (jgrandelli) [inline Example/Pods/Target Support Files/Pods-URBNSwiftyConvenience_Tests/Pods-URBNSwiftyConvenience_Tests-acknowledgements.markdown]: Mobile team email

**Comment by ryang1428** [comment]
some conflicts as well

File: Example/Pods/Target Support Files/Pods-URBNSwiftyConvenience_Tests/Pods-URBNSwiftyConvenience_Tests-acknowledgements.markdown (hunk 1 above)
Comment (codecaffeine) [reply Example/Pods/Target Support Files/Pods-URBNSwiftyConvenience_Tests/Pods-URBNSwiftyConvenience_Tests-acknowledgements.markdown]: Fixed

**Comment by codecaffeine** [comment]
Alright @jgrandelli. Conflicts fixed and comments addressed.

**Pull request merged**